templates/admin.php

```diff
@@ -20,14 +20,13 @@
 	<input type="text" id="allowed_domains" name="allowed_domains" value=<?php p($_['allowed']);?>>
 	</p>
 	<p>
-	<em><?php p($l->t('Enter a semicolon-separated list of allowed domains. Example: owncloud.com;github.com'));?></em>
+	<em><?php p($l->t('Enter a semicolon-separated list of allowed domains. Example: nextcloud.com;example.com'));?></em>
 	</p>
 
-	<p>
-	<label for="admin_approval_required"><?php p($l->t('Require admin approval?')); ?>
-	<input type="checkbox" id="admin_approval_required" name="admin_approval_required" <?php if ($_['approval_required'] === "yes") {
+	<div style="margin-top: 10px;">
```

Comment:
Instead of the margin-top on a div, you can just put the checkbox+label inside a `<p>`. :)

Reply:
No, that had no spacing at all 🙈